### DISCUSSION TITLE
Mayby a bug in total consumption

### DISCUSSION THREAD

**Discussion by brijaze** [UtilitechAS/amsreader-firmware#484] [General]
After upgrade to 2.2.9 the total consumption count on the frontpage is wrong.
Atleast at my installation 
![image](https://user-images.githubusercontent.com/32090690/224856713-00f65cad-7e17-4c61-b456-a984f9cf1fff.jpeg)

Used to read 80000+kWh 

Ideas?

**Comment by gskjold** [comment]
This is probably a result of your meter only reporting accumulated values at the top of the hour. The counter is probably in sync by now

**Reply by brijaze** [reply]
You where right. Thanks for quick reply.

Br.
Brian
